ui/app/src/elements/dialogs/create-nh-dialog.ts

```diff
@@ -7,9 +7,10 @@ import { SlInput, SlTooltip} from '@scoped-elements/shoelace';
 import { matrixContext } from "../../context";
 import { MatrixStore } from "../../matrix-store";
 import { NHComponentShoelace, NHDialog, NHSelectAvatar } from "@neighbourhoods/design-system-components";
+import { b64images } from "@neighbourhoods/design-system-styles";
 import { InferType, object, string } from "yup";
 
-const NH_DEFAULT_LOGO = "PHN2ZyB3aWR0aD0iODgiIGhlaWdodD0iODgiIHZpZXdCb3g9IjAgMCA4OCA4OCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHJlY3Qgd2lkdGg9Ijg4IiBoZWlnaHQ9Ijg4IiBmaWxsPSIjMTgxNTFCIi8+CjxyZWN0IHdpZHRoPSI4OCIgaGVpZ2h0PSI4OCIgZmlsbD0iIzI1MUYyOCIvPgo8cGF0aCBkPSJNMjYuNTAxMiA1Mi4zNDc1QzI2LjUwMTIgNTIuMzQ3NSAyNi41MDEyIDcwIDkuMDAxNjUgNzBWMzQuNjk1MkM5LjAwMTY1IDM0LjY5NTIgOS4wMDE2NSAxNy4wNDI3IDI2LjUwMTIgMTcuMDQyN0MyNi41MDEyIDE3LjA0MjcgMjYuNTAxMiAzNC42OTUxIDQ0LjAwMDggMzQuNjk1MUM2MS41MDA0IDM0LjY5NTIgNjEuNTAwNCA1Mi4zNDc2IDYxLjUwMDQgNTIuMzQ3NlYxNy4wNDI3QzYxLjUwMDQgMTcuMDQyNyA3OSAxNy4wNDI3IDc5IDM0LjY5NTFWNTIuMzQ3NUM3OSA1Mi4zNDc1IDc5IDY5Ljk5OTkgNjEuNTAwNCA3MEw0NC4wMDA4IDcwQzQ0LjAwMDggNzAgNDQuMDAwOCA1Mi4zNDc1IDI2LjUwMTIgNTIuMzQ3NVoiIGZpbGw9InVybCgjcGFpbnQwX3JhZGlhbF8xMTYzXzYxMCkiLz4KPHBhdGggZD0iTTY0LjIzMzEgMTcuMDAyN0M2Mi41OTI1IDE3LjAwNzEgNjEuNDk4NyAxNy4wMjA1IDYxLjQ5ODcgMTcuMDQyN0M2MS40OTg3IDM0LjY5NSA0My45OTkyIDM0LjY5NTIgNDMuOTk5MiAzNC42OTUyQzQzLjk5OTIgMzQuNjk1MiA0My45OTkyIDE3LjA0MjYgMjYuNDk5NiAxNy4wNDI3QzIwLjQxOTIgMTcuMDQyNyAxNi40NTM5IDE5LjE3NTUgMTMuODY0OCAyMS45NTY4QzExLjExMDkgMjQuNTY4NiA5IDI4LjU2NjcgOSAzNC42OTUyVjcwQzkuMDY2MTQgNzAgOS4xMzAwNyA2OS45OTggOS4xOTU3NyA2OS45OTc2QzkuMTk4MzMgNjkuOTk3NSA5LjIwNDU4IDY5Ljk5NzcgOS4yMDg5OSA2OS45OTc2QzE1LjE3MSA2OS45NDkxIDE5LjA3NzMgNjcuODM3MiAyMS42MzY1IDY1LjA4OEMyNC4zOTAzIDYyLjQ3NjIgMjYuNTAxMyA1OC40NzU5IDI2LjUwMTMgNTIuMzQ3Nkg2MS41MDA0QzU1LjQyIDUyLjM0NzYgNTEuNDU0OCA1NC40ODAzIDQ4Ljg2NTYgNTcuMjYxNkM0Ni4xMTE4IDU5Ljg3MzUgNDQuMDAwOCA2My44NzE1IDQ0LjAwMDggNzBINjEuMjYzNUg2MS41MDAzQzc4Ljk5OTggNzAgNzguOTk5OCA1Mi4zNDc2IDc4Ljk5OTggNTIuMzQ3NlY1Mi4yOTQ2VjMyLjM0MTVWMTcuMDQyOEM3OC45OTk4IDE3LjA0MjggNjkuMTU0OCAxNi45ODcxIDY0LjIzMzEgMTcuMDAyN1oiIGZpbGw9InVybCgjcGFpbnQxX3JhZGlhbF8xMTYzXzYxMCkiLz4KPGRlZnM+CjxyYWRpYWxHcmFkaWVudCBpZD0icGFpbnQwX3JhZGlhbF8xMTYzXzYxMCIgY3g9IjAiIGN5PSIwIiByPSIxIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgZ3JhZGllbnRUcmFuc2Zvcm09InRyYW5zbGF0ZSg2MS41MDA0IDUyLjM0NzYpIHJvdGF0ZSgtNDkuMzgyNykgc2NhbGUoNDUuODUgNjcuNzg5KSI+CjxzdG9wIHN0b3AtY29sb3I9IndoaXRlIiBzdG9wLW9wYWNpdHk9IjAuOTk2MDc4Ii8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0id2hpdGUiIHN0b3Atb3BhY2l0eT0iMCIvPgo8L3JhZGlhbEdyYWRpZW50Pgo8cmFkaWFsR3JhZGllbnQgaWQ9InBhaW50MV9yYWRpYWxfMTE2M182MTAiIGN4PSIwIiBjeT0iMCIgcj0iMSIgZ3JhZGllbnRVbml0cz0idXNlclNwYWNlT25Vc2UiIGdyYWRpZW50VHJhbnNmb3JtPSJ0cmFuc2xhdGUoMjYuNTAwOCAxNy4wNDMxKSByb3RhdGUoMTAzLjM4Mikgc2NhbGUoNzAuMDQ1OCA4Ny44MTY0KSI+CjxzdG9wIHN0b3AtY29sb3I9IndoaXRlIiBzdG9wLW9wYWNpdHk9IjAuOTk2MDc4Ii8+CjxzdG9wIG9mZnNldD0iMSIgc3RvcC1jb2xvcj0id2hpdGUiIHN0b3Atb3BhY2l0eT0iMCIvPgo8L3JhZGlhbEdyYWRpZW50Pgo8L2RlZnM+Cjwvc3ZnPgo="
```

Comment:
Probably considered a housekeeping thing, but it would be nice to have binary assets stored in binary format on disk and brought into the JavaScript environment via some build process or import loader. This would allow us to separate their change histories to not interfere with text diffs, and easily view them with image manipulation programs.
I'm also not sure that this format is a best practise (what is it, PNG data?); maybe we should be rethinking raster graphics and dealing in SVG imagery.

Reply:
@pospi There are plans to have a load of exportable SVG assets allowing the changing of icon colours based upon the surrounding text color, but the design team has the task and they have a lot on their plate right now.

So consider current handling of all of these image assets as a bit of a temporary workaround. This logo is now being exported from design-system-styles just so there isn't some hard coded binary in the launcher, at least.

Reply:
So, things like this could be based off [@media (prefers-color-scheme: dark)](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme). What we need to know is if a specific theme is dark or not. I'm not so sure if the css vars can actually cross the SVG document boundary. I haven't checked that out ever. There's also a more in depth example of this kind of thing with the Github themes over here: https://github.com/nikolalsvk/nikolalsvk

But I'll create a separate ticket for that and the design team.

Reply:
Oh, I suppose the color-scheme can just be set to the name of the theme. So we'll need a build process for our SVGs based off the theme color variables. That'll be fun.

Reply:
> I'm not so sure if the css vars can actually cross the SVG document boundary.

From memory they can!

Reply:
> Oh, I suppose the color-scheme can just be set to the name of the theme. So we'll need a build process for our SVGs based off the theme color variables. That'll be fun.

They aren't monotone by any chance, are they? Styling `fill` is a neat way of being able to do alternates... also an option to split out layers as separate SVGs and overlay / colour them separately.